result/bgpv4.yaml

```diff
@@ -46,22 +43,10 @@ components:
           description: >-
             The name of a configured BGPv4 device.
           type: string
-        sessions_total:
+        session_state:
           description: >-
-            Total number of session
-          type: integer
-        sessions_up:
-          description: >-
-            Sessions are in active state
-          type: integer
-        sessions_down:
-          description: >-
-            Sessions are not active state
-          type: integer
-        sessions_not_started:
-          description: >-
-            Sessions not able to start due to some internal issue
-          type: integer
+            Session state as up or down
+          type: string
```

Comment:
this should have enum values "up" and "down"

Reply:
updated